orchestrator-salt/src/main/resources/salt-common/salt/monitoring/systemd/cdp-prometheus.service.j2

```diff
@@ -9,7 +9,7 @@ After=network-online.target
 Type=simple
 User=root
 Group=root
-ExecStart=/opt/cdp-prometheus/prometheus \
+ExecStart=/bin/sh -c "/opt/cdp-prometheus/prometheus \
```

Comment:
why is this change required?

Reply:
The redirection of the stdout and stderr at the end of the command made the ExecStart command fail, and the service unable to start, so that's why I had to wrap it.

Reply:
please check systemd manual, according to this: https://stackoverflow.com/a/54232531 there is a more sophisticated way for redirecting output

Reply:
The solution suggested there only works with v236+ versions of systemd. If I remember correctly only v217 is available on our vms.

Reply:
I double checked, and systemd v219 cannot use the StandardOutput field to log directly into a file. I indicated this in the commit description.